### PR TITLE
Remove graphite metrics

### PR DESCRIPTION
**Pull request by frshwtr**

### CODE REVIEW

File: lib/middleware/sanitize-errors.js [hunk 1]
```diff
@@ -8,14 +8,11 @@ module.exports.formatErrorMessage = formatErrorMessage;
...
 function sanitizeErrors() {
 	return (error, request, response, next) => {
-		const {metrics} = request.app.ft;
-
 		// Format the error message if required
 		error.message = module.exports.formatErrorMessage(error);
 
 		// Compilation errors use the non-standard 560 status
 		if (error instanceof CompileError || error.type === 'CompileError') {
-			metrics.count('servererrors.compile', 1);
```
Comment (frshwtr) [inline lib/middleware/sanitize-errors.js:18]: All of these metrics can be tracked by monitoring the status code for the given endpoint. They are no longer necessary at this level.

File: lib/middleware/archive.js [hunk 2]
```diff
@@ -37,22 +36,18 @@ module.exports = function (app) {
 			}
 
 			if(s3Result.response && s3Result.response.status === 404) {
-				metrics.count('archive.404', 1);
```
Comment (frshwtr) [inline lib/middleware/archive.js:40]: Equally these can also be tracked with auto instrumentation.

Comment (notlee) [reply lib/middleware/archive.js:40]: How?

Comment (frshwtr) [reply lib/middleware/archive.js:40]: > How?

As the error code is sent through to be the response of this request, auto instrumentation will report on. We can create dashboard views that look at this specific endpoint and count the responses with the 400/200/500 codes etc.

Comment (notlee) [reply lib/middleware/archive.js:40]: 👍 Somehow I wasn't thinking that this is all on the one endpoint.

**Review by notlee — APPROVED** [review]

**Pull request merged**